<?php
namespace App\Controller;

use App\Kernel;
use Symfony\Bundle\TwigBundle\Controller\ExceptionController;

class CustomExceptionController extends ExceptionController
{
    /**
     * Handle exception
     *
     * On this app level only two possible actions: in debug mode throw exception further,
     * otherwise show the stub page for the user. No business logic here!
     *
     * @param Kernel     $kernel
     * @param \Throwable $e
     * @throws \Throwable
     */
    private function processException(Kernel $kernel, \Throwable $e): void
    {
        if ($kernel->isDebug()) {
            throw $e;
        }

        //$this->render('/api_error');
    }
}
